<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainHorizontalLayout">
    <item>
     <widget class="QListWidget" name="listFilters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightColumnLayout">
      <item>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderTimeline">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPlotPlaceholder">
        <property name="text">
         <string>WYKRES (PlotWidget)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupDisplayOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="title">
         <string>Display options</string>
        </property>
        <layout class="QVBoxLayout" name="groupBoxLayout">
         <item>
          <layout class="QHBoxLayout" name="scaleLayout">
           <item>
            <widget class="QSpinBox" name="spinBoxScaleX">
             <property name="prefix">
              <string>X: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxScaleY">
             <property name="prefix">
              <string>Y: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxLineStyle">
           <item>
            <property name="text">
             <string>Continuous line</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dotted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Intermittent</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxGrid">
           <property name="text">
            <string>Turn on the grid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonApplyOptions">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPolaczenie">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionPolaczZSerwerem"/>
    <addaction name="actionRozlacz"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuKonfiguracjaWtyczek">
    <property name="title">
     <string>Plugin configuration</string>
    </property>
    <addaction name="actionDodajFiltr"/>
    <addaction name="actionUsunFiltr"/>
    <addaction name="actionOdswiezFiltry"/>
   </widget>
   <addaction name="menuPolaczenie"/>
   <addaction name="menuKonfiguracjaWtyczek"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPolaczZSerwerem">
   <property name="text">
    <string>Connect to server</string>
   </property>
  </action>
  <action name="actionRozlacz">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDodajFiltr">
   <property name="text">
    <string>Add filter</string>
   </property>
   <property name="toolTip">
    <string>Za≈Çaduj filtr</string>
   </property>
  </action>
  <action name="actionOdswiezFiltry">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionUsunFiltr">
   <property name="text">
    <string>Remove filter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
